### PR TITLE
Feature/ikdt 611 role group defect

### PR DESCRIPTION
**Pull request by jsteinhafel**
For the defect IKDT-611 where user was not able to see additional options in axiom section.

### CODE REVIEW

**Review by rbradley813 — APPROVED** [review]
Looks good!

File: rules/src/main/java/dev/ikm/komet/rules/annotated/AxiomFocusedRules.java [hunk 1]
```diff
@@ -60,8 +60,7 @@ public void axiomIsNotDefinitionRoot(ObservationRecord $observation) {
             @MethodPredicate(method = "isAxiomFocused", args = {"$observation"}),
             @MethodPredicate(method = "isDefinitionRoot", args = {"$observation"})
     })
-    public void axiomIsDefinitionRoot(ObservationRecord $observation,
-                                      RhsContext ctx) {
+    public void axiomIsDefinitionRoot(ObservationRecord $observation) {
```
Comment (carldea) [inline rules/src/main/java/dev/ikm/komet/rules/annotated/AxiomFocusedRules.java:63]: Question: Does the method in line 61 referencing this method? If so should they match?
```java
@MethodPredicate(method = "isDefinitionRoot" ...

public void axiomIsDefinitionRoot( ...
```

Comment (rbradley813) [reply rules/src/main/java/dev/ikm/komet/rules/annotated/AxiomFocusedRules.java:63]: @carldea, the method in line 61 references the method on line 125 of RuleBase.java (method signature provided below)
`public boolean isDefinitionRoot(...`

Comment (carldea) [reply rules/src/main/java/dev/ikm/komet/rules/annotated/AxiomFocusedRules.java:63]: Excellent! Makes sense. 👍🏼

**Review by carldea — APPROVED** [review]
Looks good to me!

**Pull request merged**